src/store/compression_lz4.rs

```diff
@@ -0,0 +1,19 @@
+extern crate lz4;
```

Comment:
The goal (of putting the extern crate call here rather than lib.rs) being that we only "include the crate" in the library if that feature was enabled. Correct?

Reply:
yes... Alternative I could have added a `#[cfg()]` thing before hte statement in lib.rs.